controllers.js

```diff
@@ -71,6 +72,20 @@ const patchArticleByArticleID = (req, res) => {
     }
   });
 };
+const deleteCommentsByCommentID = (req, res) => {
+  const id = req.params.comment_id;
+  removeCommentById(id).then((data) => {
+    if (data.rowCount < 1) {
+      return res.status(404).send({ message: "comment_id does not exist" });
+    } else if (data.code == "22P02") {
+      return res
+        .status(404)
+        .send({ message: "invalid data type for comment_id" });
+    } else {
+      return res.status(204).send();
+    }
```

Comment:
Some of this logic is very repeatable (e.g. error code 22P02) and might be better handled with a less specific error message in the app - refer to these: https://l2c.northcoders.com/courses/sd-notes/back-end#sectionId=error-handling,step=intro